MAUI/ImageEditor/reset.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Reset image .NET MAUI Image Editor control | Syncfusion
+description: Learn about reset support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
```

Comment:
Description: Learn about reset support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.